### PR TITLE
Fix for Issue #3

### PR DESCRIPTION
**Pull request by immackay**

### CODE REVIEW

File: zipline_poloniex/bundle.py [hunk 1]
```diff
@@ -207,13 +219,4 @@ def day(self):
...
-    '.test_poloniex',
-    create_bundle(
-        [Pairs.usdt_eth],
-        pd.Timestamp('2016-01-01', tz='utc'),
-        pd.Timestamp('2016-01-31', tz='utc'),
-    ),
-    calendar_name='POLONIEX',
-    minutes_per_day=24*60
-)
+# Deleted the register function as the built in zipline register function plays better with other data bundles
```
Comment (FlorianWilhelm) [inline zipline_poloniex/bundle.py]: This I don't understand. Basically `register` just registers a name for the data bundle. Why is this example `.test_poloniex` a problem here?

**Review by FlorianWilhelm — COMMENTED** [review]
That's really interesting. Do you know why splitting it into 8 hour blocks solves the issue? Be already limit the number of requests to specific number of requests per seconds but do we also need to drop the connection sometimes?

File: zipline_poloniex/bundle.py (hunk 1 above)
Comment (immackay) [reply zipline_poloniex/bundle.py]: I think I may have misinterpreted this. I initially thought it conflicted with ziplines default register function, but I now realize that it shouldn't. I had done these edits a while ago.

**Comment by immackay** [comment]
The 8 hour blocks just ensures that it stays under 50000 trades. The usdt_btc pair has a few days this year that have over 100000 trades each, which conflicts with the original code.

**Comment by immackay** [comment]
Hi Florian, not sure if you saw I pushed a final version of this fix with your suggestion implemented.

**Comment by FlorianWilhelm** [comment]
@immac636 thanks for pinging me again and your PR :-) I like it, would you be interested in becoming a collaborator in this project?

**Comment by immackay** [comment]
Sure, I'm likely to keep using this and tinkering with it. I can maintain it if you like.

**Comment by FlorianWilhelm** [comment]
@immackay , cool! Welcome on board! Feel free to commit improvements and ping me if you want to have something released on PyPI.

**Pull request merged**